originals/b.yaml

```
@@ -146,6 +146,17 @@
     genres:
     - Arcade
 
+- name: 'Battle Isle 2220: Shadow of the Emperor'
```

```suggestion
- name: Battle Isle 2220
```
let's use the shorter name

---

yes, I think septerra core should use the shorter name